addon/component-managers/sparkles.ts

```diff
@@ -43,7 +43,9 @@ export default class SparklesComponentManager {
   }
 
   didCreateComponent(component: CreateComponentResult) {
-    component.didInsertElement();
+    if (getOwner(this).lookup('-environment:main').isInteractive) {
```

Comment:
I don't love this solution is there a better way @rwjblue ?

Reply:
This can't change after the component manager is created, so lets look it up once in the constructor and save it off as `this.isInteractive` (that way we don't have to do an owner.lookup per instantiated component).